src/index.js

```diff
@@ -141,6 +154,14 @@ class TransportEmailPlugin {
     this._unset()
   }
 
+  test ({to}) {
+    return this._sendEmail({
+      to,
+      subject: 'Configuration test',
```

Comment:
Subject should be more explicit: *[Xen Orchestra] Test of transport-email plugin*.